packages/perseus/src/widgets/matrix/score-matrix.test.ts

```diff
@@ -171,6 +171,32 @@ describe("scoreMatrix", () => {
         expect(result).toHaveInvalidInput();
     });
 
+    it("treats nulls in the rubric as zeroes", () => {
+        // Arrange
+        const rubric: PerseusMatrixRubric = {
+            answers: [
+                // @ts-expect-error - Type 'null' is not assignable to type 'number'.
+                [null, 1, 2],
+                [3, 4, 5],
+                [6, 7, 8],
+            ],
+        };
+
+        const userInput: PerseusMatrixUserInput = {
+            answers: [
+                [0, 1, 2],
+                [3, 4, 5],
+                [6, 7, 8],
+            ],
+        };
+
+        // Act
+        const result = scoreMatrix(userInput, rubric, mockStrings);
+
+        // Assert
+        expect(result).toHaveBeenAnsweredCorrectly();
+    });
```

Comment:
This is a characterization test for the existing scoring logic, showing that converting `null` to `0` in the matrix data preserves behavior.